models/migrations/v111.go

```diff
@@ -385,8 +382,15 @@ func addBranchProtectionCanPushAndEnableWhitelist(x *xorm.Engine) error {
 
 	// Find latest review of each user in each pull request, and set official field if appropriate
 	reviews := []*Review{}
-	var page int64
-	for page = 0; page <= totalPages; page++ {
+
```

Comment:
could you remove these migrations changes from this Pr as they are in a different PR